packages/authentication/types/allAuth.ts

```
@@ -0,0 +1,713 @@
+  // TODO
+}
+
+export interface SignUpRequest {
+  // name?: string
+  email: string
+  password: string
+}
+
+export interface SignUpResponse {
+  // TODO
+}
```

Biome

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)